src/main/java/net/helpscout/api/utils/ParamsUtils.java

```diff
@@ -66,6 +69,19 @@ public static String setFields(String url, List<String> fields) {
         return params;
     }
 
+    public static Map<String, String> getSearchParamsMap(String query, String sortField, String sortOrder, Integer page) throws ApiException {
+        Map<String, String> params = new HashMap<String, String>();
+        params.put("query", query);
+        if (sortField != null && sortField.trim().length() > 0) {
```

Comment:
There are two blocks of code that do the same -> check if items is not null and its trimmed length is > 0.

If you first construct a map with all the values upfront, you can then filter the null or empty values in one step.